src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -3421,7 +3456,16 @@ type Route implements Node {
     bikesAllowed: BikesAllowed
 
     """List of patterns which operate on this route"""
-    patterns: [Pattern]
+    patterns(
+        """
+        Filter patterns by the service dates they operate on.
+
+        **Note**: A service date is a technical term useful for transit planning purposes and might not
+        correspond to a how a passenger thinks of a calendar date. For example, a night bus running
+        on Sunday morning at 1am to 3am, might have the previous Saturday's service date.
+        """
+        serviceDates: ServiceDateFilterInput
```

Comment:
Same consideration here as well. What if we at some point to have the "ids" filter here as well.

Reply:
`ids` is deprecated now.